awx/main/utils/common.py

```diff
@@ -506,6 +507,8 @@ def get_model_for_type(type):
         ct_type = get_type_for_model(ct_model)
         if type == ct_type:
             return ct_model
+    else:
+        raise DatabaseError('"{}" is not a valid AWX model.'.format(type))
```

Comment:
@ryanpetrello I did a `for-else` here

Reply:
![giphy](https://user-images.githubusercontent.com/214912/32119917-b114463a-bb24-11e7-8464-fe453426e2b1.gif)